<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="MacBook-Pro" name="com.qa.tests.LoginTest" tests="4" failures="0" timestamp="11 Sep 2020 05:58:22 GMT" time="0.003" errors="1">
  <testcase name="invalidUserName" time="0.000" classname="com.qa.tests.LoginTest">
    <skipped/>
  </testcase> <!-- invalidUserName -->
  <testcase name="invalidPassword" time="0.000" classname="com.qa.tests.LoginTest">
    <skipped/>
  </testcase> <!-- invalidPassword -->
  <testcase name="successfulLogin" time="0.000" classname="com.qa.tests.LoginTest">
    <skipped/>
  </testcase> <!-- successfulLogin -->
  <testcase name="beforeClass" time="0.003" classname="com.qa.tests.LoginTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.qa.BaseTest.closeApp(BaseTest.java:352)
	at com.qa.tests.LoginTest.beforeClass(LoginTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- beforeClass -->
</testsuite> <!-- com.qa.tests.LoginTest -->
